core/trino-spi/src/test/java/io/trino/spi/type/TestSqlTimeWithTimeZone.java

```diff
@@ -15,14 +15,66 @@
 
 import org.testng.annotations.Test;
 
+import static io.trino.spi.type.SqlTimeWithTimeZone.newInstance;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestSqlTimeWithTimeZone
 {
+    @Test
+    public void testBaseline()
+    {
+        assertThat(newInstance(0, 0, 1).toString()).isEqualTo("00:00:00+00:01");
+        assertThat(newInstance(1, 0, 1).toString()).isEqualTo("00:00:00.0+00:01");
+        assertThat(newInstance(2, 0, 1).toString()).isEqualTo("00:00:00.00+00:01");
+        assertThat(newInstance(3, 0, 1).toString()).isEqualTo("00:00:00.000+00:01");
+        assertThat(newInstance(4, 0, 1).toString()).isEqualTo("00:00:00.0000+00:01");
+        assertThat(newInstance(5, 0, 1).toString()).isEqualTo("00:00:00.00000+00:01");
+        assertThat(newInstance(6, 0, 1).toString()).isEqualTo("00:00:00.000000+00:01");
+        assertThat(newInstance(7, 0, 1).toString()).isEqualTo("00:00:00.0000000+00:01");
+        assertThat(newInstance(8, 0, 1).toString()).isEqualTo("00:00:00.00000000+00:01");
+        assertThat(newInstance(9, 0, 1).toString()).isEqualTo("00:00:00.000000000+00:01");
+        assertThat(newInstance(10, 0, 1).toString()).isEqualTo("00:00:00.0000000000+00:01");
+        assertThat(newInstance(11, 0, 1).toString()).isEqualTo("00:00:00.00000000000+00:01");
+        assertThat(newInstance(12, 0, 1).toString()).isEqualTo("00:00:00.000000000000+00:01");
```

Comment:
Don't convert to String before comparing. Compare directly against instances created with `SqlTimeWithTimeZone.newInstance`

Reply:
I agree in case of all other tests except here we don't apply any transformation, so we have to compare against another representation.

Reply:
Ah, yes, I meant to put the comment in the methods below. For this one, rename the test to `testToString`, as that's what's being tested here.

Also, this test is unrelated to the fix in this commit, so move it to a separate commit.